raspberry_pi_experiments/raspberry_pi_experiment_readme.md

```diff
@@ -0,0 +1,9 @@
+## Model power consumption benchmark guide
+### Experiment Setup
+1. Download and install the usb serial from `http://www.ftdichip.com/Drivers/VCP.htm`.
+2. Plug the rpi into the Wattsup Pro and adjust the mode to be current watt mode.
```

Comment:
Please use full name Raspberry Pi here and everywhere in the doc or define abbreviation.